exchange_calendars/xkrx_holidays.py

```diff
@@ -0,0 +1,1231 @@
+        "2013-09-20",
+        "2013-10-03",
+        "2013-10-09",
+        "2013-12-25",
+        "2013-12-31",
+        "2014-01-01",
+        "2014-01-30",
+        "2014-01-31",
+        "2014-05-01",
+        "2014-05-05",
+        "2014-05-06",
+        "2014-06-04",
+        "2014-06-06",
+        "2014-08-15",
+        "2014-09-08",
+        "2014-09-09",
+        "2014-09-10",
+        "2014-10-03",
+        "2014-10-09",
+        "2014-12-25",
+        "2014-12-31",
+        "2015-01-01",
+        "2015-02-18",
+        "2015-02-19",
+        "2015-02-20",
+        "2015-05-01",
+        "2015-05-05",
+        "2015-05-25",
+        "2015-08-14",
+        "2015-09-28",
+        "2015-09-29",
+        "2015-10-09",
+        "2015-12-25",
+        "2015-12-31",
+        "2016-01-01",
+        "2016-02-08",
+        "2016-02-09",
+        "2016-02-10",
+        "2016-03-01",
+        "2016-04-13",
+        "2016-05-05",
+        "2016-05-06",
+        "2016-06-06",
+        "2016-08-15",
+        "2016-09-14",
+        "2016-09-15",
+        "2016-09-16",
+        "2016-10-03",
+        "2016-12-30",
+        "2017-01-27",
+        "2017-01-30",
+        "2017-03-01",
+        "2017-05-01",
+        "2017-05-03",
+        "2017-05-05",
+        "2017-05-09",
+        "2017-06-06",
+        "2017-08-15",
+        "2017-10-02",
+        "2017-10-03",
+        "2017-10-04",
+        "2017-10-05",
+        "2017-10-06",
+        "2017-10-09",
+        "2017-12-25",
+        "2017-12-29",
+        "2018-01-01",
+        "2018-02-15",
+        "2018-02-16",
+        "2018-03-01",
+        "2018-05-01",
+        "2018-05-07",
+        "2018-05-22",
+        "2018-06-06",
+        "2018-06-13",
+        "2018-08-15",
+        "2018-09-24",
+        "2018-09-25",
+        "2018-09-26",
+        "2018-10-03",
+        "2018-10-09",
+        "2018-12-25",
+        "2018-12-31",
+        "2019-01-01",
+        "2019-02-04",
+        "2019-02-05",
+        "2019-02-06",
+        "2019-03-01",
+        "2019-05-01",
+        "2019-05-06",
+        "2019-06-06",
+        "2019-08-15",
+        "2019-09-12",
+        "2019-09-13",
+        "2019-10-03",
+        "2019-10-09",
+        "2019-12-25",
+        "2019-12-31",
+        "2020-01-01",
+        "2020-01-24",
+        "2020-01-27",
+        "2020-04-15",
+        "2020-04-30",
+        "2020-05-01",
+        "2020-05-05",
+        "2020-08-17",
+        "2020-09-30",
+        "2020-10-01",
+        "2020-10-02",
+        "2020-10-09",
+        "2020-12-25",
+        "2020-12-31",
+        "2021-01-01",
+        "2021-02-11",
+        "2021-02-12",
+        "2021-03-01",
+        "2021-05-05",
+        "2021-05-19",
+        "2021-09-20",
+        "2021-09-21",
+        "2021-09-22",
+        "2021-12-31",
+    ]
+)
+
+
+# Automatically generated holidays using /etc/update_xkrx_holidays.py script.
+# Note that there are some missing holidays compared to the original holidays.
```

Comment:
Any idea why they're missing?

Reply:
Not much actually.

The following page offers holidays ranging from current year to year 2009:
http://global.krx.co.kr/contents/GLB/05/0501/0501110000/GLB0501110000.jsp
And it seems that those holidays are obviously valid with no missing holidays.

While it might seem that we can only check the holidays until year 2009 in the page,
that page's underlying API even gives responses with years older than 2009, the limitation in UI.
So I just blindly dumped the responses from that API as much as I can get and combined them with the existing ones.

I expected that it had to be the single source of truth but just turned out that it was not, with some missing holidays.
Since this is not an desirable usage in their perspective, I think we cannot expect any of integrity in data beyond year 2009.